### DISCUSSION TITLE
Workflow not running for pull requests made by bots

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#25786] [Actions]
<p>I have the following workflow setup:</p>
<pre><code class="lang-auto">name: 🔨 Build Mudlet
on:
  push:
    branches: [master, development]
  pull_request:
  workflow_dispatch:
  schedule:
    - cron: '0 2 * * *'
</code></pre>
<p>When a bot makes the PR, Github no longer runs my workflow - why? It used to before. <a href="https://github.com/Mudlet/Mudlet/pull/5025" rel="noopener nofollow ugc">Example here</a>.</p>
<p>Appreciate any assistance in this.</p>

**Comment by lirantal** [comment]
<p>Hi <a class="mention" href="/vadi2">@vadi2</a></p>
<p>I looked at the repository and it looks to be resolved now.<br>
Let me know if there’s still an ongoing issue there that I can help with.</p>

**Comment by vadi2** [comment]
<p>I’m afraid it’s still an issue, see this PR for example - <a href="https://github.com/Mudlet/Mudlet/pull/5296" class="inline-onebox" rel="noopener nofollow ugc">Update text for translation in Crowdin by github-actions · Pull Request #5296 · Mudlet/Mudlet · GitHub</a></p>
<p>I’ve just been force approving the PRs manually which is not ideal since I have to toggle push protection every time for it.</p>

**Comment by vadi2** [comment]
<p>Any way of solving this?</p>

**Comment by jsoref** [comment, marked as answer]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181093272-0ea1ec8b-ca72-4566-839e-b46c2df798af.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow" target="_blank" rel="noopener nofollow ugc">Automatic token authentication - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<blockquote>
<p>When you use the repository’s <code>GITHUB_TOKEN</code> to perform tasks on behalf of the GitHub Actions app, events triggered by the <code>GITHUB_TOKEN</code> will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository’s <code>GITHUB_TOKEN</code> , a new workflow will not run even when the repository contains a workflow configured to run when <code>push</code> events occur.</p>
</blockquote>
<p>If you need to trigger events, you’ll need to set a PAT, see:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#granting-additional-permissions">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#granting-additional-permissions" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181093273-481c4500-5494-4bf5-b0d4-20ad049949e6.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#granting-additional-permissions" target="_blank" rel="noopener nofollow ugc">Automatic token authentication - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
